source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -264,11 +264,14 @@ class Filter : public Logger::Loggable<Logger::Id::filter>,
   Http::FilterTrailersStatus encodeTrailers(Http::ResponseTrailerMap& trailers) override;
   Http::FilterMetadataStatus encodeMetadata(Http::MetadataMap& trailers) override;
   void setEncoderFilterCallbacks(Http::StreamEncoderFilterCallbacks& callbacks) override;
+  void setStartTime(absl::optional<MonotonicTime> time) { start_time_ = time; };
+  absl::optional<MonotonicTime> getStartTime() { return start_time_; };
```

Comment:
See my comments in the tests, but I don't think we want to add these public methods.